### PR TITLE
add yurt-manager docs

### PR DESCRIPTION
**Pull request by rambohe-ch**

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/yurt-manager.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+NodePool 根据特定的节点属性（如地区、CPU架构、云提供商等）将节点池的概念进行抽象化，以便在池级别统一管理节点。
+
+我们习惯于通过不同的 Kubernetes 标签对节点进行分组和管理，但随着节点和标签数量的增加，节点的操作和维护（如批量配置调度策略、污点等）将变得越来越复杂，如下图所示：
+![img](../../../../../static/img/nodepool1.png)
+
+nodepool 控制器/webhook 可以从节点池的角度管理不同边缘区域中的节点，如下图所示：
+
+![img](../../../../../static/img/nodepool2.png)
+
+### poolcoordinatorcert 控制器
```
Comment (luc99hen) [inline i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/yurt-manager.md:53]: 1.7 is missing

File: docs/core-concepts/yurt-manager.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+![img](../../static/img/nodepool1.png)
+
+`nodepool` controller/webhook can manage the nodes in different edge regions from the perspective of node pool,  as shown in the following figure:
+
+![img](../../static/img/nodepool2.png)
+
+### 1.7 `poolcoordinatorcert` controller
+
+The `poolcoordinatorcert` controller is responsible for preparing certificates and kubeconfig file for the pool-coordinator component. All certificates and kubeconfig files are stored as Secret resources withing the system.
```
Comment (luc99hen) [inline docs/core-concepts/yurt-manager.md:62]:
```suggestion
The `poolcoordinatorcert` controller is responsible for preparing certificates and kubeconfig file for the pool-coordinator component. All certificates and kubeconfig files are stored as Secret resources within the system.
```

**Review by luc99hen — CHANGES_REQUESTED** [review]

File: i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/yurt-manager.md (hunk 1 above)
Comment (rambohe-ch) [reply i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/yurt-manager.md]: fxied

File: docs/core-concepts/yurt-manager.md (hunk 2 above)
Comment (rambohe-ch) [reply docs/core-concepts/yurt-manager.md]: fixed

**Review by luc99hen — APPROVED** [review]

**Pull request merged**